paket.dependencies

```diff
@@ -20,6 +20,7 @@ nuget Microsoft.Build  copy_local:false
 nuget Microsoft.Build.Framework copy_local:false
 nuget Microsoft.Build.Utilities.Core copy_local:false
 nuget Microsoft.Build.Tasks.Core copy_local: false
+nuget NuGet.Frameworks
```

Comment:
I think we still need this set as copy local false like the MSBuild deps, so that we load the assembly from the SDK when we use msbuild via the SDK. It's used by msbuild internally and if we ship a version that is out of sync with the version shipped in the SDK and used by Msbuild then projects stop loading.